src/epd_driver/epd_driver.c

```diff
@@ -40,13 +41,73 @@ void epd_draw_vline(int x, int y, int length, uint8_t color,
   }
 }
 
+EpdRect _rotated_area(uint16_t x, uint16_t y, uint16_t w, uint16_t h) {
+  // If partial update uses full screen do not rotate anything
+  if (x != 0 && y != 0 && EPD_WIDTH != w && EPD_HEIGHT != h) {
+    switch (epd_get_rotation())
+    {
+      // 0 landscape: Leave it as is
+      // 1 90 ° right portrait
+      case 1:
```

Comment:
Same here, using enums for the orientation would be nicer, since otherwise there are these magic numbers and you have to remember which is which.

Reply:
I agree. But is just Nx90 clockwise, is not so hard :)

Reply:
True, but you only have to read the code once with enums to understand it, while you might have to think for 5 seconds more without them ;)